packages/common/src/hooks/useHostContext.ts

```diff
@@ -46,7 +46,7 @@ export const useHostContext = create<HostContext>(set => ({
   detailPanelRef: null,
 
   setStore: store => set(state => ({ ...state, store })),
-  store: { id: '4321dcba', name: 'Central Warehouse' },
+  store: { id: '4321dcba', code: 'Central Warehouse' },
```

Comment:
the API doesn't provide name or description, I've created an issue to include those.